lib/rows.js

```diff
@@ -81,9 +81,9 @@ Rows.prototype.query = function queryRows() {
   }
 
   let builtPlan = args[0],
-      options = (args.length === 2) ? args[1] : null;
+      options = (args.length >= 2) ? args[1] : null;
 
-  return queryRowsImpl(this, builtPlan, null, options, args[1]);
+  return queryRowsImpl(this, builtPlan, null, options, args[2]);
 };
```

Comment:
@mariusmarklogic , we might need to use args[2] instead of args[1] since args[1] is expected to be sent by plan-search.js and other test failures on jenkins